docs/spfx/update-latest-packages.md

```diff
@@ -0,0 +1,116 @@
+
+```
+npm outdated
+```
+
+The command will list the following information about the packages your project depends on. This information is looked up from the `package.json` file located in the root of your project directory and npm registry.
+
+* Current version installed in your project
+* Version requested by your project (available in `package.json`)
+* Latest version available
+
+![NPM outdated packages](../../images/npm-outdated-packages-list.png)
+
+To identify the SharePoint Framework packages, look for the package names that start with the following npm scope and prefix:
+
+```
+@microsoft/sp-
+```
+
+### Update packages
+To update one or more packages to the latest version, you will need to edit the package(s) information in the `package.json` file and then fetch the latest packages.
+
+#### Update package versions
+Open your project in your favorite code editor and locate the `package.json` file in the root of your project directory.
+
+In the `package.json` file, locate the package(s) under the `dependencies` and `devDependencies` section and update the version to the latest version available that was listed in the `npm outdated` command. For example, the image below highlights the version updates to SharePoint Framework packages, the left section referring to the old and the right section referring to the latest package versions.
+
+![Edit package versions in package.json file](../../images/npm-update-packagejson-versions.png)
+
+Once you have updated the package versions, Save the `package.json` file.
+
+#### Update packages
+Open your favorite console and navigate to the root of your project directory. Follow the instructions below to successfully update the packages to its latest version.
+
+Delete the `node_modules` folder. This is the folder where npm installs the packages locally for your project. Deleting this folder forces npm to fetch the latest and not duplicate existing packages.
+
+If you are using a Mac or a Linux environment, then run the following command:
+
+```
+rm -rf node_modules/
+```
+
+If you are using a Windows environment, then run the following command:
+
+```
+ rd /s /q node_modules/
+```
+
+Now, execute the following command to fetch the latest packages from the npm registry:
+
+```
+npm install
+```
+
+This command will create the `node_modules` folder and install all the packages your project depends on based on the information available in the `package.json` file. As we updated the file with the latest versions, npm will now have the latest packages installed. 
+
+Once the packages are installed, execute the following command to clean up any old build artifacts:
+
+```
+gulp clean
+```
+
+## Update yeoman generator for SharePoint
+Unlike the npm packages that are installed to a specific client-side project, the yeoman generator for SharePoint is installed globally in your machine.
```

Comment:
The scenario of installing the generator locally to a project will be looked into post GA as commented in this issue #302 . So lets update the setup doc to reflect what model is supported for now.